exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,4 @@
-const name = " Daniel  ";
-
-console.log(message);
+const newName = " Daniel  ".trim();
```

Comment:
You don't need to delete the original name variable, you can call the trim method on that variable and you get the same result, this would be a cleaner implementation